python/dgl/nn/pytorch/conv/gatconv.py

```diff
@@ -298,6 +304,9 @@ def forward(self, graph, feat, get_attention=False):
             if self.res_fc is not None:
                 resval = self.res_fc(h_dst).view(h_dst.shape[0], -1, self._out_feats)
                 rst = rst + resval
+            # bias
+            if self.bias is not None:
+                rst = rst + self.bias.view(1, -1, self._out_feats)
```

Comment:
```suggestion
                rst = rst + self.bias.view(1, self._num_heads, self._out_feats)
```
This looks more clear to me.